x-pack/plugins/observability_solution/slo/server/services/historical_summary_client.test.ts

```diff
@@ -46,7 +46,7 @@ const generateEsResponseForRollingSLO = (slo: SLODefinition, overridedRange?: Da
     ? rollingDurationInDays + overridedRangeInDays
     : rollingDurationInDays * 2;
   const numberOfBuckets = fullDuration * bucketsPerDay;
-  const startDay = moment().subtract(fullDuration, 'day').startOf('day');
+  const startDay = moment().subtract(fullDuration, 'day').startOf('minute');
```

Comment:
@kdelemme Now that you changed to `startOf('minute')`, maybe you would like to change the name of the current `startDay` variable? Would `startMinute` make sense?

Reply:
I guess startRange would be even better